### PR TITLE
Bump sdk version aaaaand use `clone_from`

### PR DESCRIPTION
**Pull request by Narcha**
what the title says

### CODE REVIEW

**Comment by oleflb** [comment]
> what the title says

It does so much more than the title suggested :disappointed:

**Review by oleflb — APPROVED** [review]

**Comment by Narcha** [comment]
Fixed it :)

File: crates/constants/src/lib.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ pub const HULA_DBUS_SERVICE: &str = "org.hulks.hula";
 pub const HULA_SOCKET_PATH: &str = "/tmp/hula";
 pub const OS_RELEASE_PATH: &str = "/etc/os-release";
 pub const OS_VERSION: &str = "7.4.2";
-pub const SDK_VERSION: &str = "7.4.0";
+pub const SDK_VERSION: &str = "7.4.2";
```
Comment (schmidma) [inline crates/constants/src/lib.rs:17]: Why was this done? This leads to all HULKs downloading the newest patch version when running pepsi?! And using bumping this without also bumping the pepsi version also doesn't make sense... Please explain ;)

Comment (Narcha) [reply crates/constants/src/lib.rs:17]: This was necessary after updating the CI image. Without it, our CI couldn't find the SDK (it was looking for version 7.4.2, the latest meta-nao release). Knoelle knows the details of this.

**Pull request merged**